lisa/tests/staging/sched_android.py

```diff
@@ -236,4 +236,78 @@ def test_stune_frequency(self, freq_margin_pct=10) -> ResultBundle:
         }
         return self._merge_res_bundles(res_bundles)
 
+
+class SchedTunePlacementItem(SchedTuneItemBase):
+    """
+    Runs a tiny RT-App task marked 'prefer_idle' at a given boost level and
+    tests if it was placed on big-enough CPUs.
+    """
+
+    ftrace_conf = FtraceConf({
```

Comment:
hand-specified `ftrace_conf` is not necessary anymore since #824 as long as all tests methods are decorated with the required events.